client/js/widgets/widget.js

```diff
@@ -1504,6 +1504,10 @@ export class Widget extends StateManaged {
         } else if(collection = getCollection(a.collection)) {
           if(collections[collection].length) {
             await shuffleWidgets(collections[collection]);
+            await w(collections[collection].map(i=>i.get('parent')), async holder=>{
+              if(holder.get('type') == 'holder')
+                await holder.updateAfterShuffle();
+            });
```

Comment:
This sounds like a nice feature but I don't think we can just change the behavior like this for existing games. I have no idea though why this code is in `SORT` but not in `SHUFFLE`.

Maybe we should make it optional in both with the same parameter?

Reply:
Good idea. Will do, @ArnoldSmith86 !

Reply:
Are your shuffling the holder or are you just updating the holder in case at least one card in in a holder?

Reply:
@RaphaelAlvez I should've provided a bit more background info. I was trying to give each player's HAND a shuffle button, but it doesn't update the shuffled result. @ArnoldSmith86 suggested [a workaround](https://virtualtabletop.io/l67n) to sort it with an empty key right after shuffle, just to trigger that `updateAfterShuffle` in SORT, which for some reason is not in SHUFFLE.
 
This PR adds that back in.